Dockerfile

```diff
@@ -24,4 +24,4 @@ COPY --from=builder /build/bin/combo .
 
 EXPOSE 8080
 
-CMD ["/bin/combo"]
\ No newline at end of file
+CMD ["/bin/combo", "run"]
```

Comment:
I think we should hold off changing the Dockerfile here and instead, handle this at the Deployment level by specify the command(s) explicitly. This should help alleviate any failures during the build workflow when attempting to run the newly built container image as the newly build container won't be expecting a cluster to be running (like the run command will need).

Reply:
Updated!